### PR TITLE
📅  Add datepicker component

### PR DESCRIPTION
**Pull request by sdevalapurkar-bcgov**
## Overview

- Add datepicker component

Screen Shot 2020-08-13 at 10 17 01 AM

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "shared-components",
-  "version": "0.4.10",
+  "version": "0.5.0",
```
Comment (akroon3r) [inline package.json:3]: how are we determining the difference between patch and minor on this repo?

Comment (sdevalapurkar-bcgov) [reply package.json:3]: brand new component added or new feature is like going from 4 -> 5
updates to existing is 4.1 -> 4.2 for example 
is how we have been doing it so far

Comment (akroon3r) [reply package.json:3]: so if you update an 0.4.X feature in 0.5.X, you increment to 0.5.X+1?

Comment (sdevalapurkar-bcgov) [reply package.json:3]: yeah!

Comment (akroon3r) [reply package.json:3]: cool cool

**Pull request merged**